### PR TITLE
Create hero section

### PR DESCRIPTION
**Pull request by CamilingJS**
### What changes have been made?

- Depends on #4.

- Implemented Hero Section based off the wireframe

### What tickets does this PR associate with? (Link to ticket) 

- No ticket was created for this section

### How has the UI changed? (Screenshot or recording, before or after)

- Before:
image

- After:
image

### Does it work on mobile? (Screenshot for different screen sizes)

- Yes, it is mobile responsive
image
image
image

### Where there changes made to the schema?

- No changes to database schema

### Why were these changes made?

- Designs were change from a copy paste component to customized Tailwind based off a wireframe

### What tests have been added?

- No

### How was it implemented, and why was this implementation chosen?

- Tailwind utility classes

### If a new npm package was added, why was it chosen and why was it necessary?

- No new npm packages

### What alternatives were considered, and why weren't they used?

- Inline CSS was considered but decided to utilize Tailwind

### Documentation of challenges that were faced and how they were fixed

- Challenges were due to lack of knowledge of Tailwind classes, but we were able to research via google and ChatGPT

### CODE REVIEW

File: app/root.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ import globalStyles from "./styles/global.css"
 
 
 
-import TopNav from "~/components/topNav";
+import TopNav from "~/components/topNav";
```
Comment (eiffelwong1) [inline app/root.tsx:19]: run `npx eslint . --fix` to fix minor formatting/style issues

File: tailwind.config.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@ export default {
   theme: {
     extend: {},
   },
+
```
Comment (eiffelwong1) [inline tailwind.config.ts:8]: Nit comment: best to remove this white line, and make the PR cleaner.

**Review by eiffelwong1 — DISMISSED** [review]
looks good to me

File: app/components/HeroSection.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import heroImg from "../imgs/Hero-Image.png"
```
Comment (andrelandgraf) [inline app/components/HeroSection.tsx:1]: I recommend naming all files in `kebab-case` to match npm package naming! Any idea how we could enforce this with a GitHub action? :)

File: app/components/HeroSection.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+export default function HeroSection() {
+  return (
+    <section className="bg-[#E4EFF0] pt-10 pb-40">
```
Comment (andrelandgraf) [inline app/components/HeroSection.tsx:5]: Is this a color we will use a lot (`bg-[#E4EFF0])`)? If so, we can add it to the `tailwind.config` file and give it a nice name like "primary" or "warm" and use it like this: `bg-warm`

File: app/components/HeroSection.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import heroImg from "../imgs/Hero-Image.png"
+
+export default function HeroSection() {
```
Comment (andrelandgraf) [inline app/components/HeroSection.tsx:3]: Personal preference: Let's avoid default exports and instead export everything as none default. That way, we can easily export several components from the same file!

**Review by andrelandgraf — COMMENTED** [review]
Looks great! Left some nit comments.

File: app/components/HeroSection.tsx (hunk 5 above)
Comment (ShowMale) [reply app/components/HeroSection.tsx:3]: It sounds great to me!

**Review by andrelandgraf — APPROVED** [review]

**Review by eiffelwong1 — APPROVED** [review]
lgtm

**Pull request merged**